spec/services/search_spec.rb

```diff
@@ -86,39 +75,24 @@
       let!(:multi_match_page) {
         create(:page,
                title: 'multimatch page',
-               widgets: [matching_widget, petition_widget],
                language: language,
                campaign: campaign,
                tags: [tag])
       }
-      let(:new_params) {
+      let(:multi_match_params) {
         { search: {
-            content_search: test_text,
+            content_search: 'multimatch page',
             tags: [tag.id],
-            widget_type: ['PetitionWidget'],
             language: language.id
           }
         }
       }
-      let(:new_page_searcher) { Search::PageSearcher.new(new_params) }
-      it 'finds a page that matches the search query by tags, language, widget type and text content' do
+      let(:new_page_searcher) { Search::PageSearcher.new(multi_match_params) }
+      it 'finds a page that matches the search query by tags, language and text content' do
         expect(new_page_searcher.search).to match_array([multi_match_page])
       end
     end
 
   end
 
-  describe 'WidgetSearcher' do
-
-    it 'finds the matching widget based on the search text' do
-      expect(Search::WidgetSearcher.text_widget_search(test_text)).to eq([matching_widget])
-    end
-
-    it 'finds widgets that match the specified widget type' do
-      expect(Search::WidgetSearcher.widget_type_search(['PetitionWidget'])).to match_array(
-                [petition_widget])
-    end
-
-  end
-
 end
```

Comment:
I would say that especially since the private methods themselves aren't tested, it makes sense to test the search method itself with all kinds of queries that users might throw at it. Regardless of whether the method is private or that you're implicitly testing it, it makes sense to test a bunch of expected behavior, at least for things like positive / negative and and / or.